composeApp/src/commonMain/composeResources/values/strings-common.xml

```diff
@@ -76,8 +76,8 @@
     <string name="Settings_AutomatedTesting_RunAutomatically_ChargingOnly">Only while charging</string>
     <string name="Settings_AutomatedTesting_RunAutomatically_Footer">By enabling automatic testing, OONI Probe tests will run automatically multiple times per day. Your test results will automatically get published on OONI Explorer: https://explorer.ooni.org/ \n\nImportant: If you have a VPN enabled, OONI Probe will not run tests automatically. Please turn off your VPN for automated OONI Probe testing. Learn more: https://ooni.org/support/faq/#can-i-run-ooni-probe-over-a-vpn</string>
 
-    <string name="Settings_Websites_MaxRuntimeEnabled">Limit test duration</string>
-    <string name="Settings_Websites_MaxRuntime">Test duration</string>
+    <string name="Settings_Websites_MaxRuntimeEnabled">Limit Websites test duration</string>
+    <string name="Settings_Websites_MaxRuntime">Maximum Websites test duration</string>
```

Comment:
These should preferably be new strings since there are existing translations we dont want these to be confused for.